frontend/src/components/ScheduledEventsRunHistoryTableCard/utils/index.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+import * as localForage from 'localforage';
+import { ScheduledEventRunHistory } from '../../../types/scheduledEvents';
+import { api, localForageKeys } from '../../../utils';
+
+const BASEPATH = api.routes.FETCH_RUN_INSTANCES;
+
+export const fetchRunHistory = async (eventId: number): Promise<any> => {
+  const token = await localForage.getItem<string>(localForageKeys.API_KEY);
+  const headers = {
+    'Content-Type': 'application/json',
+    Authorization: `token ${token}`,
+  };
+
+  return await axios(`${BASEPATH.replace('id', eventId.toString())}`, { headers });
+};
+
+export const getScheduledEventsByPage = (
+  currentPage: number,
+  data: ScheduledEventRunHistory[],
+): ScheduledEventRunHistory[] => {
+  const begin = (currentPage - 1) * 5;
+  const end = begin + 5;
```

Comment:
We call these magic numbers ... I'm guessing 5 refers to the limit? What happens when the limit changes? You'll have to update several lines ... so either pass in this value as a parameter, or use the constant... I've seen it somewhere already.